packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/toolbar/toolbar-overrides.tsx

```diff
@@ -16,7 +16,7 @@ export type ToolbarOverrideType =
   | 'raw'
   | 'embed'
   | 'table'
-  | 'hr';
+  | 'hr'
```

Comment:
```suggestion
  | 'hr';
```

Purely to reduce conflict errors.